src/main/java/org/jabref/gui/preftabs/PreviewPrefsTab.java

```diff
@@ -127,13 +130,17 @@ private void setupLogic() {
                 testPane.setEntry(TestEntry.getTestEntry());
                 JFXPanel container = CustomJFXPanel.wrap(new Scene(testPane));
                 container.setPreferredSize(new Dimension(800, 350));
+
+                //TODO: Add dialog for showing pane
```

Comment:
This should be easy using https://github.com/JabRef/jabref/blob/f7d96784dda0ee0d9adefd43b5a6803058284500/src/main/java/org/jabref/gui/DialogService.java#L114-L120